kim_convergence/ase/equilibration.py

```
@@ -0,0 +1,218 @@
+        ...     property_name="max_force",
+        ...     extractors={"max_force": get_max_force},
+        ... )
+        >>> result = run_ase_equilibration(sampler, relative_accuracy=0.1)
+    """
+    # Build kwargs for run_length_control
+    # Prevent override of get_trajectory (sampler) as it's critical to function operation
+    if "get_trajectory" in kwargs:
+        raise ValueError(
+            "Cannot override 'get_trajectory' parameter. "
+            "The sampler is automatically set from the provided sampler argument."
+        )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)